config/postgresql/config.go

```diff
@@ -19,7 +19,9 @@ package postgresql
 import (
 	"context"
 	"fmt"
+	"strconv"
 
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
```

Comment:
We can organize imports as follows:
<stdlib imports>
<3rd party imports>
<crossplane-runtime imports>
<local imports>